index.html

```diff
@@ -13,5 +13,6 @@ <h2 id="role-title"></h2>
     <p id="role-description"></p>
     <span id="error-message"></span>
 </main>
+<script src="scripts/main.js">Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.</script>
```

Comment:
Je hebt in je javascript al de error message gedefinieerd, en hier doet hij volgens mij niets. Dus deze kan weg (dan werkt hij nog steeds)